scripts/update-feather-icons.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs')
+const https = require('https')
+const path = require('path')
+
+https.get('https://unpkg.com/feather-icons@4.19.0/dist/icons.json', resp => {
```

Comment:
```suggestion
https.get('https://unpkg.com/feather-icons/dist/icons.json', resp => {
```

Apparently we can omit the version and it'll auto-redirect 👌 This solution works for me, thanks for chatting through it!